target/start-mailserver.sh

```diff
@@ -49,6 +49,9 @@ DEFAULT_VARS["SPAMASSASSIN_SPAM_TO_INBOX"]="${SPAMASSASSIN_SPAM_TO_INBOX:=0}"
 DEFAULT_VARS["MOVE_SPAM_TO_JUNK"]="${MOVE_SPAM_TO_JUNK:=1}"
 DEFAULT_VARS["VIRUSMAILS_DELETE_DELAY"]="${VIRUSMAILS_DELETE_DELAY:=7}"
 DEFAULT_VARS["NETWORK_INTERFACE"]="${NETWORK_INTERFACE:="eth0"}"
+DEFAULT_VARS["ENABLE_SHARED_INBOX"]="${DOVECOT_ENABLE_INBOX_SHARING:=0}"
```

Comment:
Why are different variable names used here?
`DEFAULT_VARS["DOVECOT_ENABLE_INBOX_SHARING"]="${DOVECOT_ENABLE_INBOX_SHARING:=0}"` would be more consistent compared to the rest.

Reply:
@matejak what's up with this?

Reply:
My bad, this is done. Sorry for the noise.